docs/source/model_doc/fsmt.rst

```diff
@@ -0,0 +1,53 @@
+FSMT
+----------------------------------------------------
+**DISCLAIMER:** If you see something strange,
+file a `Github Issue <https://github.com/huggingface/transformers/issues/new?assignees=&labels=&template=bug-report.md&title>`__ and assign
+@stas00.
+
+Overview
+~~~~~~~~~~~~~~~~~~~~~
+
+This model is a porting of a transformer model for translation, discussed in `this paper <https://arxiv.org/abs/1907.06616>`__:
```

Comment:
I would follow the template of other models here:

"FSMT (FairSeq MachineTranslation) models were introduce in `Facebook FAIR's WMT19 News Translation Task Submission <this paper <https://arxiv.org/abs/1907.06616>`__ by xxx, yyy and zzz.

The abstract of the paper is the following:

*abstract*

The original code can be found `here <https://github.com/pytorch/fairseq/tree/master/examples/wmt19>`__.
"""